toolchain/parse/state.def

```diff
@@ -250,8 +250,7 @@ CARBON_PARSE_STATE(CodeBlockFinish)
 //   (state done)
 CARBON_PARSE_STATE(DeclNameAndParams)
 
-// Handles regular parameters such as `(...)` for the general declaration case.
-// Only used after implicit parameters.
+// Handles implicit parameters such as `[...]` for the general declaration case.
```

Comment:
Is the example below incorrect? Because this looks the original comment is correct. Instead, on like 266, maybe you want "after _explicit_ parameters".

Reply:
Note too, this is document as _after_ the parameters, not handling the parameters themselves.

Reply:
BTW, feel free to edit as needed. I'm just suggesting after looking at the implementation. If you want something like "completed handling", just make the sentence and I can review because I'm not sure how to phrase with that unless you mean like "Only used after implicit parameters completed handling".

Reply:
Yes, ok I agree those make sense comparing with the implementation, thanks. Using them verbatim.